concourse/pipelines/windows-image-build.yaml

```diff
@@ -1042,6 +1049,51 @@ jobs:
       job: "build-windows-2012r2-core"
       result_state: "failure"
       start_timestamp: ((.:start-timestamp-ms))
+# sql-2012-enterprise-windows-2012-r2-dc
+- name: build-sql-2012-enterprise-windows-2012-r2-dc
+  plan:
+  - get: compute-image-tools
+    passed: [build-windows-2012r2]
```

Comment:
Since we're referencing the source image as source_image_project: "bct-prod-images", we wouldn't want this to kick off after the build, but instead after the publish to testing as that is when the image would be present in the bct-prod-images project.

Reply:
Once this is updated to `` you can also change the publish-to-testing-2012r2 task so that it automatically triggers. 

Then when a windows build is kicked off the publish to testing happens immediately after the build and then after that step the SQL images would start.

Reply:
Sounds good. Will update soon. Thank you.